### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by pressplay21** [OBAViJEST/rain#11]
No updates for months. Is this project still alive?

**Comment by raipat** [comment]
no, it's not dead. I forked the repo and a community team will continue to develop RAIN. So yes, the project is alive :)
